app/components/NoSuggestionsPanel/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Icon from '../Icon'
+import { IntlMixin } from 'react-intl'
+
+/**
+ * Generic panel showing an icon and message, to
+ * use when there are no suggestions to display.
+ */
+let NoSuggestionsPanel = React.createClass({
```

Comment:
The name is NoSuggestionsPanel but it can be used for any message display. Maybe rename it?